pkg/maps/encrypt/encrypt.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encrypt
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+// EncryptKey is the context ID for the encryption session
+type EncryptKey struct {
+	key uint32 `align:"ctx"`
```

Comment:
nit: I'd expect the BPF side of this to show up in `bpf/bpf_alignchecker.c` for this align tag to mean something. That said, With a one-field struct I'm not sure whether there's a way for the alignment to get mixed up..